tests/framework/validators/NumberValidatorTest.php

```diff
@@ -309,6 +309,12 @@ public function testValidateResource()
         $model->attr_number = $fp;
         $val->validateAttribute($model, 'attr_number');
         $this->assertTrue($model->hasErrors('attr_number'));
+        
+        // the check is here for HHVM that
+        // was losing handler for unknown reason
+        if (is_resource($fp)) {
+            fclose($fp);
+        }
```

Comment:
IMO if you want the most correct solution, you must skip this test for HHVM because of idea of the test for HHVM is broken (`$fp` isn't valid resource and the test need a valid resource).

Reply:
In general, this call of `fclose` has no any effect in any case. `$fp` is a local var, so a resource with no more references to it is detected automatically, and it is freed by the garbage collector. Isn't it?

Reply:
Skipping test completely on HHVM sounds bad for me because stable version works well with it.